### PR TITLE
fix AdminEventModal (admin v1) to show the correct errors for the 'room' field

### PR DESCRIPTION
**Pull request by malaikaaiyar**
Uses room validation for the room field instead of host validation

(note: there is no validation schema for the room; should it be a required field or something?)

### CODE REVIEW

**Comment by 0xdevalias** [comment]
> (note: there is no validation schema for the room; should it be a required field or something?)

Off the top of my head, it’s not required since events can be made that aren’t linked to rooms.

File: src/pages/Admin/AdminEventModal.tsx [hunk 1]
```diff
@@ -214,8 +214,8 @@ const AdminEventModal: React.FunctionComponent<PropsType> = ({
                 ref={register}
                 value={roomName}
               />
-              {errors.host && (
-                <span className="input-error">{errors.host.message}</span>
+              {errors.room && (
```
Comment (sunny-viktoryia) [inline src/pages/Admin/AdminEventModal.tsx:217]: Looks like `room` does not have any validation requirements. So we can remove this block `{errors.room && ...}` fully. 
But ideally would be to add a validation rule allowing only existent room names. But probably it's out of scope of this PR.

**Review by sunny-viktoryia — APPROVED** [review]
Approving this but if you have capacity would be awesome if you could add validation for `room` field (see my comment). 🙃

File: src/pages/Admin/AdminEventModal.tsx (hunk 1 above)
Comment (malaikaaiyar) [reply src/pages/Admin/AdminEventModal.tsx:217]: I'll merge this and make a new PR for that

**Pull request merged**